Makefile

```diff
@@ -15,3 +15,12 @@ test_no_proxy:
 test_proxy:
 	@echo "Running all tests with proxy process"
 	CODEJAIL_PROXY=1 nosetests --with-xunit --xunit-file reports/nosetests-proxy.xml
+
+
+export CUSTOM_COMPILE_COMMAND = make upgrade
```

Comment:
Shouldn't this be like this: https://github.com/edx/edx-platform/blob/master/Makefile#L68

Reply:
fixed